include/exadg/solvers_and_preconditioners/multigrid/multigrid_preconditioner_base.cpp

```diff
@@ -420,124 +327,67 @@ create_geometric_coarsening_sequence(Triangulation<dim, spacedim> const & fine_t
   });
 
   // for a single level nothing has to be done
-  if(fine_triangulation_in.n_global_levels() > 1)
-  {
-    auto const fine_triangulation =
-      dynamic_cast<parallel::distributed::Triangulation<dim, spacedim> const *>(
-        &fine_triangulation_in);
-
-    Assert(fine_triangulation, ExcNotImplemented());
```

Comment:
Right now, I unconditionally delete the old code; if we still want to support dealii-9.3 (the last release version), which might be a good idea, we should wrap this into `#ifdef` clauses and switch between the two variants. (Noting that the newer one is faster and takes less memory.)

Reply:
I thought dealii-9.3 is the new release? So did these changes come into dealii shortly after this year's release, or do you mean support of dealii-9.2?

Reply:
We branched off the release branch before so these features are not part of 9.3.